tests/scripts/run-xunit-perf.cmd

```diff
@@ -15,6 +15,7 @@ setlocal
   set TEST_CONFIG=Release
   set IS_SCENARIO_TEST=
   set USAGE_DISPLAYED=
```

Comment:
Then, after parsing the command line arguments passed to the script, we should check:
```cmd
  if defined BENCHVIEW_PATH (
    if not exist "%BENCHVIEW_PATH%" (
      call :print_error BenchView path: "%BENCHVIEW_PATH%" was specified, but it does not exist.
      exit /b 1
    )
  )
```